# Optimizes the trajectory and footholds to reach a given goal state

StateLin3d goal_state
---